evmcc/evmcc_test.go

```diff
@@ -684,6 +686,7 @@ Vc4foA7mruwjI8sEng==
 
 					Expect(stub.SetEventCallCount()).To(Equal(1))
 					setEventName, setEventPayload := stub.SetEventArgsForCall(0)
+					// Expect(setEventName).To(Equal(hex.EncodeToString(contractAddress[0:8])))
 					Expect(setEventName).To(Equal(string(contractAddress[0:8])))
```

Comment:
Expect(utf8.ValidString(setEventName)).To(BeTrue())

Reply:
That isn't quite the error we are trying to catch though. The commented out expect also is a check to make sure it is a utf 8 valid string. The utf8 validity though in addition with checking the value of the name itself may be a good check  I am trying to cause https://github.com/hyperledger/fabric-chaincode-evm/blob/master/eventmanager/eventmanager.go#L42 to fail because that is the error that is being thrown in the error that @aklenik found. With no modification of the code & just changes to the cert and nonce we should be failing at L671.

Reply:
This 671?
```
					contractAddress, err = crypto.AddressFromHexString(string(res.Payload))
```
so the string() call is what should blow up?

I couldn't tell what the comment was for. I don't see how commented code could check for utf8.